spec/support/authentication_steps.rb

```diff
@@ -0,0 +1,9 @@
+module AuthenticationSteps
+  def and_i_sign_in(user)
+    sign_in_page = SignInPage.new
+    sign_in_page.load
+    sign_in_page.email_field.set user.email
+    sign_in_page.password_field.set 'Password1!'
```

Comment:
Hi @benbarnett, as it's only for testing purposes it seems fine as there isn't any risk associated with it.